web/app/routes/mainRoutes.js

```diff
@@ -1,24 +1,36 @@
 var express = require("express");
 var router = express.Router();
 var mongoose = require("mongoose");
+
+// For listing of supported status codes in this package
+// https://www.npmjs.com/package/http-status-codes
+var statusCodes = require("http-status-codes");
```

Comment:
Ah, I was thinking that there was a lib for status codes. I was just gonna straight up create my own static vars for em. It's great that there is an already existing one!